.github/workflows/swift-toolchain.yml

```
@@ -1809,45 +1817,52 @@ jobs:
           certutil -decode $CertificatePath $PFXPath
           Echo CERTIFICATE=$PFXPath | Out-File -FilePath ${env:GITHUB_ENV} -Encoding utf8 -Append
 
-      - name: Package SDK
+      - name: Package SDK (also builds runtime msm)
```

I don't think that the extended name helps and makes it harder to see at the smaller sizes.

```suggestion
      - name: Package SDK
```